taskana-adapter-camunda-system-connector/src/main/java/pro/taskana/adapter/camunda/outbox/rest/CamundaTaskEvent.java

```diff
@@ -13,6 +13,7 @@
   private String blockedUntil;
   private String error;
   private String camundaTaskId;
+  private String systemEngineIdentifier;
```

Comment:
Isn't this a duplicate to the class in the model package?

Reply:
It is, but they are in two different modules -> taska-adapter-camunda-system-connector and the outbox REST...same goes for the "ReferencedTask" in 2 different modules...we decided to keep them as duplicates and see if more of these duplications occure before putting them in e.g. a common module

Reply:
ok